unit-tests-jest/numbers.test.ts

```diff
@@ -0,0 +1,49 @@
+// import { isExportDeclaration } from 'typescript';
+import { evenNumbers, toDollars, multiplyBy, divideBy } from './numbers';
+describe('return even nums', () => {
+  it('description', () => {
```

Comment:
Then here you would say what you are testing:
`it('returns even numbers', () => {`
So it reads something like, "it returns even numbers".